Documentation/network/index.rst

```diff
@@ -0,0 +1,127 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _network_root:
+
+Networking
+==========
+
+.. _multi host networking:
+
+Networking Concepts
+-------------------
+.. toctree::
+   :maxdepth: 2
+   :glob:
+
+   networking/routing
+   networking/ipam/index
+   networking/masquerading
+   networking/fragmentation
+
+Kubernetes Integration
+----------------------
+.. toctree::
+   :maxdepth: 1
+   :glob:
+
+   kubernetes/intro
+   kubernetes/concepts
+   kubernetes/requirements
+   kubernetes/configuration
+   kubernetes/policy
+   kubernetes/kubeproxy-free
+   kubernetes/ciliumendpoint
+   kubernetes/ciliumendpointslice
+   kubernetes/compatibility
+   kubernetes/troubleshooting
+
+Kubernetes Networking
+---------------------
+.. toctree::
+   :maxdepth: 2
+   :glob:
+
+   bandwidth-manager
+   kata
+   ipam
+   local-redirect-policy
+   ciliumendpointslice
```

Comment:
I think we need to double-check what is in "Kubernetes Integration" and what is in "Kubernetes Networking". For example CiliumEndpointSlice is covered in both places and I'm wondering if that makes sense

Reply:
Resolved